<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>472</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>681</width>
    <height>472</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>681</width>
    <height>472</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pok√©mon Text Editor</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0" rowspan="2">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="indentation">
       <number>20</number>
      </property>
      <column>
       <property name="text">
        <string>INI</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="styleSheet">
       <string notr="true">QPlainTextEdit#plainTextEdit
{
	border: none;
	border-bottom: 1px solid #b9b9b9;
}</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QStatusBar" name="statusbar2">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Visible linebreaks will be ignored upon compiling text. Use &lt;span style=&quot; font-style:italic;&quot;&gt;\p&lt;/span&gt;, &lt;span style=&quot; font-style:italic;&quot;&gt;\n&lt;/span&gt; and &lt;span style=&quot; font-style:italic;&quot;&gt;\l&lt;/span&gt; instead.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QStatusBar#statusbar2
{
 	border-bottom: 1px solid #b9b9b9;
	background-color: #ffffff;	
}</string>
      </property>
      <property name="sizeGripEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>treeWidget</zorder>
   <zorder>plainTextEdit</zorder>
   <zorder>statusbar2</zorder>
   <zorder>statusbar2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_ROM"/>
    <addaction name="actionSave_ROM"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSearch_Text"/>
    <addaction name="actionCreate_specific_INI"/>
    <addaction name="separator"/>
    <addaction name="actionWrite_current_text"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_ROM">
   <property name="text">
    <string>Open ROM...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_ROM">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save ROM</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionSearch_Text">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Search Text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionCreate_specific_INI">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create specific INI</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Creates an INI specifically for your &lt;span style=&quot; font-style:italic;&quot;&gt;ROM-hack&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionWrite_current_text">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Write current text</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
